pkg/kubelet/kubelet_node_status.go

```diff
@@ -595,15 +595,17 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 			}
 		}
 
-		currentCapacity := kl.containerManager.GetCapacity()
-		if currentCapacity != nil {
-			for k, v := range currentCapacity {
-				if v1helper.IsExtendedResourceName(k) {
-					glog.V(2).Infof("Update capacity for %s to %d", k, v.Value())
-					node.Status.Capacity[k] = v
-				}
```

Comment:
Is this removing the updates that could have happened independently to "extended resources" created via node patch? Not all extended resources are device (plugin) resources, right?

Reply:
This only affects extended resources we get through kl.containerManager and currently it only covers device plugin resources. Extended resources added through node patch should be unaffected.

Reply:
Thanks for the answer.